torchvision/prototype/transforms/_misc.py

```diff
@@ -0,0 +1,48 @@
+import warnings
+from typing import Any, Dict, Sequence
+from typing import Callable
+
+import torch
+from torchvision.prototype.features import Image
+from torchvision.prototype.transforms import Transform
+
+
+class Identity(Transform):
+    """Identity transform that supports all built-in :class:`~torchvision.prototype.features.Feature`'s."""
+
+    def __init__(self):
+        super().__init__()
+        for feature_type in self._BUILTIN_FEATURE_TYPES:
+            self.register_feature_transform(feature_type, lambda input, **params: input)
+
+
+class Lambda(Transform):
+    def __new__(cls, lambd: Callable) -> Transform:  # type: ignore[misc]
+        warnings.warn("transforms.Lambda(...) is deprecated. Use transforms.Transform.from_callable(...) instead.")
+        # We need to generate a new class everytime a Lambda transform is created, since the feature transforms are
+        # registered on the class rather than on the instance. If we didn't, registering a feature transform will
+        # overwrite it on **all** Lambda transform instances.
+        return Transform.from_callable(lambd, name="Lambda")
```

Comment:
Should we even implement it? I think it might be better to just ask users to implement their own transform, no?

Reply:
IIRC, we wanted to keep BC for the transforms. If that still holds, we need it. Otherwise I'm happy to remove it.

Reply:
We can keep the old `Lambda` transform as it was originally implemented, and say that it doesn't work with the new style of transforms. This is basically how I would envision a deprecation of a transform on a new style transforms